mp/src/game/client/momentum/tf2proxy.cpp

```diff
@@ -35,7 +35,12 @@ class CBurnLevelProxy : public CResultProxy
 {
   public:
     bool Init(IMaterial *pMaterial, KeyValues *pKeyValues) { return true; }
-    void OnBind(void *pC_BaseEntity) {}
+    void OnBind(void *pC_BaseEntity)
+    {
+#ifdef _WIN32
+        m_pResult->SetFloatValue(0.0f);
+#endif
+    }
     IMaterial *GetMaterial() { return nullptr; }
```

Comment:
Upon testing this fix, windows gets a nullptr exception for the `m_pResult` pointer. Remove the `Init()` and `GetMaterial()` functions from this.